translations/zh/built-in-makers-zh.md

```diff
@@ -0,0 +1,80 @@
+内置建造器 Built In Makers
+===============
+下面是一个Fili内置指标列(metric)建造器表格。对于指标列(metric)建造器，请参考[`Configuring Metrics`][configuring metrics]。
+
+Druid聚合操作建造器
+---------------------------------------
+如下列举的是一些一级指标列(metric)制造器，它们和Druid数据聚合功能一一对应。
+[Druid aggregations][druid aggregations]。
+
+
+1. [`长整数之和建造器LongSumMaker`][长整数之和建造器LongSumMaker]
+2. [`双精度之和建造器DoubleSumMaker`][精度之和建造器DoubleSumMaker]
+3. [`长整数最大值建造器LongMaxMaker`][长整数最大值建造器LongMaxMaker]
+4. [`双精度最大值建造器DoubleMaxMaker`][双精度最大值建造器doubleMaxMaker]
+5. [`长整数最小值建造器LongMinMaker`][长整数最小值建造器LongMinMaker]
+6. [`双精度最小值建造器DoubleMinMaker`][双精度最小值建造器DoubleMinMaker]
+7. [`计数器建造器CountMaker`][计数器建造器CountMaker]<sup>[1](#countCaveat)</sup>
+
+另外，Fili的核心库支持Sketch类型和Sketch集合操作。详见
+[theta sketches in Druid][druid sketch module]。
+
+8. [`Sketch计数器建造器SketchCountMaker`][Sketch计数器建造器SketchCountMaker] -
+sketchCount与[sketch aggregation][sketch module]相关联。
+
+关于sketch的更多内容可参考http://datasketches.github.io/.
+
+Druid后聚合操作建造器
+--------------------------------------------
+
+下列这些建造器在查询上应用了一次Druid后聚合运算符。这些运算符对应Druid的后期数据聚合操作。
+[Druid post-aggregations][druid post-aggregations]
+
+1. [`运算建造器ArithmeticMaker`][运算建造器ArithmeticMaker]
+2. [`常量建造器ConstantMaker`][常量建造器ConstantMaker]
+
+Fili也支持Sketch集合操作。
+
+3. [`Sketch集合运算建造器SketchSetOperationMaker`][Sketch集合运算建造器SketchSetOperationMaker] - 是[Druid sketch module][sketch module]的一部分。
+
+我们现在并不支持源生Druid `JavaScript` 及 `HyperUnique Cardinality`后聚合器。
+Fili支持sketches而不是hyperUnique对象，因为二者功能重复。`JavaScript`后聚合器在大规模运行中会造成性能瓶颈。
```

Comment:
I'm not 100% sure if 
```which handle the same problem domain```
 in the original doc should be translated to 
```
因为二者功能重复
```

Reply:
I think this looks OK.

Reply:
I think this translation should be OK.